.gitignore

```diff
@@ -5,6 +5,9 @@
 .idea/*
 !.idea/codeStyles/
 
+sample_client/.idea/*
+!.idea/codeStyles/
```

Comment:
This one is already ignored 👀

Reply:
No. This one is not blocked, and I accidentally committed it. So to prevent this I added this line.

Reply:
I mean it is the same as line 6 👀

Reply:
O.o uff